index.html

```diff
@@ -0,0 +1,332 @@
+                            </a>
+                        </li>
+                        <li>
+                            <a href="#" class="social-network">
+                                <i class="fa fa-github"></i>
+                            </a>
+                        </li>
+                       <li>
+                        <a href="#" class="social-network">
+                            <i class="fa fa-angellist"></i>
+                        </a>
+                       </li>
+                    </ul>                
+                </div>
+                <div class="action">
+                    <button type="button" class="btn about-me-btn"><span>Get my resume</span> </button>
+                </div>
+                <ul class="skill-list">
+                    <!----<div class="frame-12"> --->
+                    <li class="frame-7">
+                        <a href="#">Languages</a>
+                        <!-----cheveron icon here------->
+                        <i class='fa fa-chevron-right'></i>
+                    </li>
+                    <!---<ul class="language-list"> --->
+                            
+                        <li>
+                            
+                                <div class="frame-9 common-ppty">
+                                    <img src="./images/Ellipse-1.png" alt="Ellipse 1(Javascript)">
+                                    <a href="#">Javascript</a>
+                                </div>
+    
+                                
+                        </li>   
+                        <li>
+                            
+                            <div class="frame-10 common-ppty">
+                                <img src="./images/ellipse-2.png" alt="Ellipse 1(HTML)">
+                                <a href="#">HTML</a>
+                            </div>
+
+                            
+                    </li>                         
+    
+                                       
+                        <li>
+                            
+                                <div class="frame-11 common-ppty">
+                                    <img src="./images/ellipse-3.png" alt="Ellipse 3(CSS)">
+                                    <a href="#">CSS</a>
+                                </div>
+    
+                                
+                        </li>
+
+                        <!-----</ul>   ----->              
+                    <!---</div> --->
+
+                    <li class="skill-and-framework">
+                        <a href="#" class="frameworks-frame-8">Frameworks</a>
+                        <i class='fa fa-chevron-right'></i> 
+                    </li>
+                    <li class="skill-and-framework">
+                        <a href="#" class="skills-frame-9">Skills</a>
+                        <i class='fa fa-chevron-right'></i> 
+                    </li>
+                </ul>
+            </div>
+    
```

Comment:
- [ ] Please kindly style this technology section to match the Figma template and remove multiple spaces

*Your design*
![5](https://user-images.githubusercontent.com/94551484/186519343-d808c1f9-b748-4441-9340-3a5f79384caf.png)

*Figma template*
![eee](https://user-images.githubusercontent.com/94551484/186519352-36fc43b0-5459-4ac3-820c-be63b9830c63.png)
.